components/compliance-service/test_data/audit_reports/2018-03-05_centos(2)-beta-nginx(f)-apache(f)-failed.json

```diff
@@ -24,118 +24,54 @@
   "chef_tags": [],
   "ipaddress": "10.3.4.5",
   "fqdn": "web-cent.example.com",
+  "run_time_limit": 1.1,
   "profiles": [
     {
-      "name": "nginx-baseline",
```

Comment:
Converted this report into a hybrid one where one of its profiles is stripped of the metadata.